Package.swift

```diff
@@ -2,8 +2,14 @@ import PackageDescription
 
 let package = Package(
   name: "SnowMan",
+  targets: [
+  	Target(name: "Dictionary"),
+  	Target(name: "SnowManLogic", dependencies: ["Dictionary"]),
+ 	Target(name: "Application", dependencies: ["SnowManLogic"]),
+ 	Target(name: "Specs", dependencies: ["SnowManLogic"]),
+  ],
   dependencies: [
-    .Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,7))
+    .Package(url: "https://github.com/zachmokahn/Swiftest.git", Version(0,1,8))
```

Comment:
change this to `.Package(url: "https://github.com/bppr/Swiftest.git", Version(0,1,8))`, I deleted my fork once there was an offical release of 0.1.8